Tests/LibJS/test-bytecode-js.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2022, David Tuin <davidot@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Bytecode/Generator.h>
+#include <LibJS/Bytecode/Interpreter.h>
+#include <LibJS/Interpreter.h>
+#include <LibJS/Runtime/VM.h>
+#include <LibJS/Script.h>
+#include <LibTest/TestCase.h>
+
```

Comment:
:thinking: can't these be done with test-js?
since the runtime doesn't run in BC mode yet, it could just grow an extra `compileToByteCode` function imo

(not really hellbent on this, feel free to ignore)

Reply:
Most of these can probably be done with `test-js` but not the multiple files loading ones and this is supposed to be somewhat temporary until we can run `test-js` with bytecode.